### PR TITLE
Radio component

### PR DESCRIPTION
**Pull request by MariaEleniT**
Done:

- Radio component.

- Test for component. All existing tests passing

- Storybook story for component.

Still to be done:

- Empty .scss file for component, as styling is still TBD.

### CODE REVIEW

File: src/components/atoms/input/Radio/Radio.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    type: string;
+    onClick: () => void;
+}
+
+const Radio: React.FC<RadioProps> = (props) => {
+    const [value, setValue] = useState("")
+
+    const buttonClicked = (e) => {
+        setValue(e.target.value)
+        console.log("Clicked")
```
Comment (bguyan) [inline src/components/atoms/input/Radio/Radio.tsx]: youll want to make sure you pull these out before you push to the repo, its useful for dev but we dont want logs getting into the codebase

File: src/components/atoms/input/Radio/Radio.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import React, { HTMLAttributes, useState } from "react";
+
+export interface RadioProps extends HTMLAttributes<HTMLElement> {
+    id?: string;
+    className?: string;
+    value: string;
+    checked: boolean;
+    type: string;
```
Comment (bguyan) [inline src/components/atoms/input/Radio/Radio.tsx]: do you need this?

File: src/index.ts [hunk 3]
```diff
@@ -1 +1,3 @@
-export * from './components/buttons/DefaultButton/DefaultButton';
\ No newline at end of file
+export * from './components/atoms/buttons/DefaultButton/DefaultButton';
+export * from './components/atoms/input/TextInput/TextInput';
```
Comment (bguyan) [inline src/index.ts]: just putting the comment about pulling out the testentry stuff here so it gets done before we can merge the code

**Review by bguyan — COMMENTED** [review]
We need to set some specific guidelines in place when it comes to committing the CSS for these but i would at bare minimum include a "wrapping" setup, so if changes are made they are made within the class for the component and should not interact with any other CSS on the page

File: src/components/atoms/input/Radio/Radio.tsx (hunk 2 above)
Comment (MariaEleniT) [reply src/components/atoms/input/Radio/Radio.tsx]: I am not sure actually, let's chat about this. Still trying to find my way around Typescript :)

**Comment by ghost** [comment]
Rebasing work, closing PR and deleting branch

**Pull request closed without merging**